### ISSUE TITLE
[Potential Feature] Database usage for Product Items/Prices etc

### ISSUE THREAD

**Issue by 2M4U** [FivePixels/Tribble#15]
Why not use something like [`mongodb`](https://npmjs.org/package/mongoose) to store what is essentially being stored here in the `config.json`

```json
 . . . 
"ITEMS_TO_SELL": "Apple,Banana,Spaghetti",
"ITEMS_PRICES": "$0.50,$1.00,$5.00",
"ITEMS_DESCRIPTIONS": "An amazing apple, Bewildering banana, Scrumptious spaghetti",
. . . 
```

Then make a command(s) for setting individual things like:

$addproduct productname productdesc productprice

Just a suggestion :)

**Comment by FivePixels** [comment]
Definitely a route to take. I don't think I'd do mongodb, but a local database is good (just like the project currently uses for storing existing ticket states). I could create a new table in the (currently) settings database to hold product info and incorporate commands as you mentioned.

I can get started on this after #13.

**Comment by FivePixels** [comment]
Also worth mentioning, changes in #13 are going to remove the COMMAND_PREFIX option since it is implementing slash commands.

**Comment by 2M4U** [comment]
I agree with the `localization` of a database for such as small user base, however if someone was to turn this project into something like a mainstream shop-network and integrate many other forms of payment (ie: Crypto-wallets or Bluesnap/Paydo) an external database would suffice on the scaling, say the end user has approx 200-500 products, it would require some modification regarding the embedment to detect if it exceeds (iirc) 2000 characters (including Product Name & Product Desc). 

> Definitely a route to take. I don't think I'd do mongodb, but a local database is good (just like the project currently uses for storing existing ticket states). I could create a new table in the (currently) settings database to h
